docs/src/intro/fetchers.md

```diff
@@ -1,38 +1,6 @@
 # Fetchers
 
-This section describes the available source code fetchers.
-
-## What are fetchers
-
-When dream2nix reads project dependencies and translates them into
-a dream2nix lockfile, it writes down the sources for all the
-dependencies: so it can fetch them later during building.
-
-Below are some examples of the sources written in the dream-lock file
-(at the end, under `sources`).
-
-```json
-"symfony/process": {
-  "v6.1.3": {
-    "rev": "a6506e99cfad7059b1ab5cab395854a0a0c21292",
-    "type": "git",
-    "url": "https://github.com/symfony/process.git"
-  }
-}
-```
-
-```json
-"fast-glob": {
-  "3.2.11": {
-    "hash": "sha512-xrO3+1bxSo3ZVHAnqzyuewYT6aMFHRAd4Kcs92MAonjwQZLsK9d0SF1IyQ3k5PoirxTW0Oe/RqFgMQ6TcNE5Ew==",
-    "type": "http",
-    "url": "https://registry.npmjs.org/fast-glob/-/fast-glob-3.2.11.tgz"
-  }
-}
-```
-
-The `type` field tells dream2nix what fetchers to invoke to get this
-source, the rest of the attributes are simply passed to the fetcher.
+This section describes the available source code fetchers. A fetcher is responsible for loading sources defined in the generic lockfile described earlier.
```

Comment:
as the lockfile description is removed; An introducing sentence is now missing. Let me know if i need to improve that.